### PR TITLE
Civ 5292 pip api

### PR DESCRIPTION
**Pull request by TatsyMaslova**
### JIRA link ###

https://tools.hmcts.net/jira/browse/CIV-5292

### Change description ###
point to real api. handle bad request and unauthorised responses

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/app/client/civilServiceClient.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ import {
   CIVIL_SERVICE_CASES_URL,
   CIVIL_SERVICE_DOWNLOAD_DOCUMENT_URL,
   CIVIL_SERVICE_FEES_RANGES,
-  CIVIL_SERVICE_SUBMIT_EVENT,
+  CIVIL_SERVICE_SUBMIT_EVENT, CIVIL_SERVICE_VALIDATE_PIN_URL,
   // CIVIL_SERVICE_VALIDATE_PIN_URL,
```
Comment (commitBlob) [inline src/main/app/client/civilServiceClient.ts:12]: can you clean this up a little bit please
put this url in the new line and remove commented out route

File: src/main/routes/features/public/firstContact/pinController.ts [hunk 2]
```diff
@@ -33,24 +33,30 @@ pinController.get(FIRST_CONTACT_PIN_URL, (req: AppRequest<{pin:string}>, res: Re
...
+      const response: AxiosResponse = await civilServiceClient.verifyPin(<AppRequest>req, req.body.pin, cookie.claimReference);
       await saveDraftClaim(response.data.id, response.data.case_data);
       cookie.claimId = response.data.id;
       cookie.pinVerified = YesNo.YES;
       res.cookie('firstContact', cookie);
       res.redirect(FIRST_CONTACT_CLAIM_SUMMARY_URL);
     }
   } catch (error) {
-    next(error);
+    const status = error.message;
```
Comment (commitBlob) [inline src/main/routes/features/public/firstContact/pinController.ts:50]: are you sure 40* statuses are going to throw an exception?
This looks very Java approach, I think what Felipe did is in the spirit of JavaScript, happy to keep it your way.

**Review by laarmada — APPROVED** [review]

File: src/main/routes/features/public/firstContact/pinController.ts (hunk 2 above)
Comment (TatsyMaslova) [reply src/main/routes/features/public/firstContact/pinController.ts:50]: yes, unfortunately i found out during testing. hence had to change the controller

File: src/main/app/client/civilServiceClient.ts (hunk 1 above)
Comment (TatsyMaslova) [reply src/main/app/client/civilServiceClient.ts]: doh!!! thank you super Maro!!!!!!!!! was doing too many things at the same time yesterday. thank you for spotting this. done

**Review by commitBlob — APPROVED** [review]

**Review by portizdev — APPROVED** [review]

**Pull request merged**